docker/Dockerfile.ca-styleguide

```diff
@@ -1,23 +1,34 @@
-FROM node:14-alpine
+FROM ruby:2.7-alpine
 
-ENV BUNDLER_VERSION 2.1.4
-# Alpine doesn't keep older patch versions, so we can only specify the minor version here
-ENV RUBY_VERSION=2.6
-ENV LANG=C.UTF-8 \
+ENV NODE_MAJOR_VERSION=14 \
+    BUNDLER_VERSION=2.1.4 \
+    LANG=C.UTF-8 \
     BUNDLE_JOBS=6 \
     BUNDLE_RETRY=3
 
-RUN apk --no-cache --update add ruby~$RUBY_VERSION ruby-dev~$RUBY_VERSION ruby-json ruby-etc gcc libc-dev libxslt-dev libxml2-dev zlib-dev build-base ruby-bigdecimal git && gem install bundler -v $BUNDLER_VERSION
+RUN apk add bash curl gcc libc-dev libxslt-dev libxml2-dev zlib-dev build-base inotify-tools && \
+    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* && \
```

Comment:
What are these two lines for? Do we need to copy them over to other containers to keep up with best practice?

Reply:
It was something mentioned online about doing the equivalent of an `apt-get clean` I also think we have/had it in public-website (the clean). Happy to remove if you want me to?

Reply:
No, that's fine. I just wasn't sure what it was for.